src/pages/learn.tsx

```diff
@@ -27,9 +18,8 @@ import LeftNavBar from "@/components/LeftNavBar"
 import InlineLink from "@/components/Link"
```

Comment:
Same with `ButtonLink`

Reply:
@pettinarip are OK then with importing from the `tailwind` directory? 

I wouldn't want to move them over to the main directory with their stories without triggering any UI tests from Chromatic, unless you think we are ready to get rid of the old story files.

Reply:
Yeah, I think we are ready to move these stories. We need to migrate them now so, no point of having the old ones.

If you update this branch with dev, we should have the stories working fine again.

Reply:
Ok, on it!

Reply:
@pettinarip Noting that my first attempt in doing this for this page threw this error:
```
Error: React.Children.only expected to receive a single React element child.
```

This point to the Radix `Slot` which is only looking for one child component. Don't know why this is inconsistent. Going to be more incremental on my second try getting the new buttons and links into prod to pinpoint the issue.

Since this is supposed to be an expected issue with `Slot`, may need to consider explicitly defining the `children` type, at least for `Button` since it will use `Slot`
```ts
children: JSX.Element // Expect only a single element, else throw
```
This would mean applying the fragment if there are multiple.

Thoughts here?